src/shared/ws2ipdef.rs

```diff
@@ -36,13 +36,14 @@ UNION!{union SOCKADDR_IN6_LH_u {
     sin6_scope_id sin6_scope_id_mut: ULONG,
     sin6_scope_struct sin6_scope_struct_mut: SCOPE_ID,
 }}
-STRUCT!{struct SOCKADDR_IN6_LH {
+STRUCT!{struct SOCKADDR_IN6 {
     sin6_family: ADDRESS_FAMILY,
     sin6_port: USHORT,
     sin6_flowinfo: ULONG,
     sin6_addr: IN6_ADDR,
     u: SOCKADDR_IN6_LH_u,
 }}
+pub type SOCKADDR_IN6_LH = SOCKADDR_IN6;
```

Comment:
According to the Windows SDK, the struct is defined as `SOCKADDR_IN6_LH` with `SOCKADDR_IN6` being a typedef to `SOCKADDR_IN6_LH`.